projects/kubernetes-sigs/kind/Makefile

```diff
@@ -39,7 +39,7 @@ VERSIONED_BASE_IMAGE=$(IMAGE_REPO)/$(KIND_BASE_IMAGE_COMPONENT):$(EKSD_KUBE_VERS
 HAS_S3_ARTIFACTS?=true
 HAS_RELEASE_BRANCHES=true
 
-PROJECT_DEPENDENCIES=eksd/kubernetes/client eksa/kubernetes-sigs/etcdadm eksd/cni-plugins eksa/kubernetes-sigs/cri-tools eksd/etcd/etcd.tar.gz
+PROJECT_DEPENDENCIES=eksd/kubernetes/client eksa/kubernetes-sigs/etcdadm eksd/cni-plugins eksa/kubernetes-sigs/cri-tools eksd/etcd/etcd.tar.gz eksa/rancher/local-path-provisioner
```

Comment:
this really should have always been here. When building locally using the docker helper not having this causes a build failure.